vmm/src/lib.rs

```diff
@@ -1049,6 +1075,10 @@ impl Vmm {
             })?
         };
 
+        if send_data_migration.local {
```

Comment:
Looks like we will need to update our vm-migration protocol for this change (e.g. the comments on the top of `vm-migration/src/protocol.rs`).